assets/js/script.js

```diff
@@ -1,97 +1,101 @@
 $(document).ready(function() {
-    var letter1Choice = "black"
-    var letter2Choice = "black"
+  var el = {
+    input:          $('#letters-input'),
+    letterTabs:     $('.letter-color-letter'),
+    letterTab1:     $('#letter1'),
+    letterTab2:     $('#letter2'),
+    lettersWrapper: $('.letters-wrapper'),
+    swatch:         $('.swatch-image-border'),
+  }
 
-    $(window).resize(function() {
-        var newLetterHeight = $(".letters-wrapper ").height()
-        $("#bagLetter1").css({ display: "block", height: newLetterHeight + "px" })
-        $("#bagLetter2").css({ display: "block", height: newLetterHeight + "px" })
+  Object.keys(el).forEach(function(key) {
+    if( !el[key].length ) { console.warn(`No element present for ${key}`); }
+  })
```

Comment:
this is just a quick check that gives us the warnings when an element we expect no longer exists in the DOM